fe/fe-core/src/main/java/com/starrocks/sql/ast/DescribeStmt.java

```diff
@@ -63,6 +64,13 @@ public class DescribeStmt extends ShowStmt {
                     .addColumn(new Column("Table", ScalarType.createVarchar(30)))
                     .build();
 
+    private static final ShowResultSetMetaData DESC_TABLE_FUNCTION_TABLE_META_DATA =
```

Comment:
Why not split this class into multiple sub-classes?
DescOlapTableStmt, DescMVStmt, DescFilesTableFuncStmt, DescCredentialStmt, ...